examples/games/game_menu.rs

```diff
@@ -349,13 +388,15 @@ mod menu {
             (Changed<Interaction>, With<Button>),
         >,
     ) {
-        for (interaction, mut image, selected) in &mut interaction_query {
-            image.color = match (*interaction, selected) {
+        for (interaction, mut color, selected) in &mut interaction_query {
```

Comment:
I think the unchanged `button_system` with `image` is more clear.

Reply:
Reverted in next commit. This was caused by a bad rebase, I think.